src/main/java/com/example/schedule/demo/service/ScheduleServiceImpl.java

```diff
@@ -54,8 +54,8 @@ public Schedule updateSchedule(Integer id, Schedule schedule) throws ScheduleNot
 
     @Override
     public void delete(Integer id, Schedule schedule) {
-        Optional<Schedule> scheduleId = this.scheduleMapper.findById(id);
-        if (scheduleId.isPresent()) {
+        Optional<Schedule> existedSchedule = this.scheduleMapper.findById(id);
```

Comment:
過去分詞よりも現在分詞のほうが違和感ないですね。
```suggestion
        Optional<Schedule> existingSchedule = this.scheduleMapper.findById(id);
```

Reply:
修正しました！
[コミットハッシュです。](https://github.com/aokiayukodesu/Schedule-list/pull/10/commits/8d4ea07dc1dbea69e6a5fa28453d283f5e3c9886)

Reply:
src/main/java/com/example/schedule/demo/service/ScheduleServiceImpl.java
内の変数の修正が漏れてますね。
IntelliJではcmd shift fでファイル内文字列検索がつかえます。
その機能を活用して漏れがないか確認しましょう。

Reply:
検索機能があるんですね！勉強になります、ありがとうございます。
変数名修正しました！[コミットハッシュです](https://github.com/aokiayukodesu/Schedule-list/pull/10/commits/d78da95f2faa17991c5101d9d57bc6212abac12f)

Reply:
OKです。